pkg/controllers/management/gke/gke_cluster_handler.go

```diff
@@ -347,17 +347,23 @@ func (e *gkeOperatorController) updateGKEClusterConfig(cluster *mgmtv3.Cluster,
 
 // generateAndSetServiceAccount uses the API endpoint and CA cert to generate a service account token. The token is then copied to the cluster status.
 func (e *gkeOperatorController) generateAndSetServiceAccount(cluster *mgmtv3.Cluster) (*mgmtv3.Cluster, error) {
-
 	ctx := context.Background()
 	ts, err := controller.GetTokenSource(ctx, e.SecretsCache, cluster.Spec.GKEConfig)
```

Comment:
This could potentially move into `getRestConfig` for consistency with the equivalent method in the EKS handler

Reply:
Done.